doc/source/programs/options/formats_vector.rst

```diff
@@ -0,0 +1,18 @@
+.. option:: --formats
+
+    List all vector formats supported by this GDAL build (read-only and
+    read-write) and exit. The format support is indicated as follows:
+
+    * ``ro`` is read-only driver
+    * ``rw`` is read or write (i.e. supports :cpp:func:`GDALDriver::CreateCopy`)
+    * ``rw+`` is read, write and update (i.e. supports :cpp:func:`GDALDriver::Create`)
+    * A ``v`` is appended for formats supporting virtual IO (``/vsimem``, ``/vsigzip``, ``/vsizip``, etc).
+    * A ``s`` is appended for formats supporting subdatasets.
+
+    The order in which drivers are listed is the one in which they are registered,
+    which determines the order in which they are successively probed when opening
+    a dataset. Most of the time, this order does not matter, but in some situations,
+    several drivers may recognize the same file. The ``-if`` option of some utilities
+    can be specified to ask to use a particular driver to open the dataset.
+    Alternativel, the :config:`GDAL_SKIP` configuration option can also be used
+    to exclude one or several drivers.
```

Comment:
Isn't if `OGR_SKIP` for vectors?

Reply:
> Isn't it `OGR_SKIP` for vectors?

They have been mostly synonymous for a long time. The only difference is the separator (space for GDAL_SKIP, comma for OGR_SKIP)

Reply:
Should that reflect homehow to corresponding parts in https://gdal.org/user/configoptions.html?

Reply:
> Should that reflect homehow to corresponding parts in https://gdal.org/user/configoptions.html?

The :config:`GDAL_SKIP`  will be rendered as a hyperlink to https://gdal.org/user/configoptions.html#driver-management